### PR TITLE
Minor improvements of Russian localization

### PR DESCRIPTION
**Pull request by CrossVas**
This PR contains some improvements of Russian localization.
The wiki section is still WIP. Sorry, it requires a lot more polishing than I expected.

### CODE REVIEW

File: src/main/resources/assets/ic2/lang/blocks_ru_ru.json [hunk 1]
```diff
@@ -1,478 +1,478 @@
...
+  "block.ic2.fuel_boiler": "Топливный котел",
+  "block.ic2.fuel_boiler_structure": "Структурный блок топливного котла",
+  "block.ic2.fusion_reactor": "Термоядерный реактор",
+  "block.ic2.generator": "Генератор",
+  "block.ic2.geothermal_generator": "Геотермальный генератор",
+  "block.ic2.glassfiber_cable": "Стекловолоконный провод",
+  "block.ic2.gold_cable": "Золотой провод",
+  "block.ic2.hover_tube": "Парящая труба",
+  "block.ic2.hyperclocked_pump": "Гипер разогнанная помпа",
+  "block.ic2.induction_furnace": "Индукционная плавильня",
```
Comment (cubicbyte) [inline src/main/resources/assets/ic2/lang/blocks_ru_ru.json:243]: This is better to be changed to Индукционная печь

File: src/main/resources/assets/ic2/lang/advancements_ru_ru.json [hunk 2]
```diff
@@ -1,86 +1,86 @@
...
+  "advancement.ic2.build_rotary_macerator": "Вращающаяся дробилка",
+  "advancement.ic2.build_rotary_macerator.desc": "Создайте роторный дробитель",
+  "advancement.ic2.build_singularity_compressor": "Сжатие сингулярности",
+  "advancement.ic2.build_singularity_compressor.desc": "Создайте сингулярный компрессор",
+  "advancement.ic2.build_teleporter": "Межпространственное искривление",
+  "advancement.ic2.build_teleporter.desc": "Создайте телепорт",
+  "advancement.ic2.build_terraformer": "Измени мир",
+  "advancement.ic2.build_terraformer.desc": "Создайте терраформер",
+  "advancement.ic2.build_vacuum_canner": "Быстрый слив",
+  "advancement.ic2.build_vacuum_canner.desc": "Создайте аакуумный консервирующий механизм",
```
Comment (cubicbyte) [inline src/main/resources/assets/ic2/lang/advancements_ru_ru.json:63]: There's a typo here

**Review by cubicbyte — CHANGES_REQUESTED** [review]
A couple of changes need to be made

**Comment by Speiger** [comment]
@CrossVas 
in the main language file this entry was added:
(en_us)
```
  "death.attack.radiation": "Radiated to death",
```

**Comment by CrossVas** [comment]
@Speiger added new entry

**Comment by CrossVas** [comment]
You think I should use the old translation?

**Pull request merged**